README.md

```diff
@@ -3,14 +3,14 @@ Multiple instance derivations inside a single macro annotation
 
 version = "0.8.1"
```

Comment:
looks like current version is 0.10.0
https://github.com/manatki/derevo/blob/19f6d16cdb475965554c14c580c064f9674f6f06/publish.sbt#L3

Reply:
Last version in sonatype repo "0.9.0" was published yesterday. I updated readme.